specification/search/data-plane/Azure.Search/stable/2023-11-01/searchservice.json

```diff
@@ -0,0 +1,10740 @@
+      },
+      "description": "Defines parameters for a search index that influence scoring in search queries."
+    },
+    "ScoringFunctionAggregation": {
+      "type": "string",
+      "enum": [
+        "sum",
+        "average",
+        "minimum",
+        "maximum",
+        "firstMatching"
+      ],
+      "x-ms-enum": {
+        "name": "ScoringFunctionAggregation",
+        "modelAsString": false,
+        "values": [
+          {
+            "value": "sum",
+            "name": "Sum",
+            "description": "Boost scores by the sum of all scoring function results."
+          },
+          {
+            "value": "average",
+            "name": "Average",
+            "description": "Boost scores by the average of all scoring function results."
+          },
+          {
+            "value": "minimum",
+            "name": "Minimum",
+            "description": "Boost scores by the minimum of all scoring function results."
+          },
+          {
+            "value": "maximum",
+            "name": "Maximum",
+            "description": "Boost scores by the maximum of all scoring function results."
+          },
+          {
+            "value": "firstMatching",
+            "name": "FirstMatching",
+            "description": "Boost scores using the first applicable scoring function in the scoring profile."
+          }
+        ]
+      },
+      "description": "Defines the aggregation function used to combine the results of all the scoring functions in a scoring profile."
+    },
+    "CorsOptions": {
+      "properties": {
+        "allowedOrigins": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The list of origins from which JavaScript code will be granted access to your index. Can contain a list of hosts of the form {protocol}://{fully-qualified-domain-name}[:{port#}], or a single '*' to allow all origins (not recommended)."
+        },
+        "maxAgeInSeconds": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The duration for which browsers should cache CORS preflight responses. Defaults to 5 minutes.",
+          "x-nullable": true
+        }
+      },
+      "required": [
+        "allowedOrigins"
+      ],
+      "externalDocs": {
+        "url": "https://docs.microsoft.com/rest/api/searchservice/Create-Index"
+      },
+      "description": "Defines options to control Cross-Origin Resource Sharing (CORS) for an index."
+    },
+    "Suggester": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the suggester."
+        },
+        "searchMode": {
+          "type": "string",
+          "enum": [
+            "analyzingInfixMatching"
+          ],
+          "x-ms-enum": {
+            "name": "SuggesterSearchMode",
+            "modelAsString": false,
+            "values": [
+  
```

Comment:
PoliCheck failres here, not sure if there's a way to suppress it

Reply:
@tjprescott?

Reply:
Per the details, for exceptions contact vsswagger@microsoft.com

Reply:
Thanks @tjprescott